app/src/test/kotlin/io/intrepid/skotlinton/testutils/TestViewModelConfiguration.kt

```diff
@@ -1,24 +1,26 @@
 package io.intrepid.skotlinton.testutils
 
-import io.intrepid.skotlinton.base.PresenterConfiguration
+import io.intrepid.skotlinton.base.ViewModelConfiguration
 import io.intrepid.skotlinton.logging.CrashReporter
 import io.intrepid.skotlinton.rest.RestApi
 import io.intrepid.skotlinton.settings.UserSettings
+import io.intrepid.skotlinton.utils.TimeProvider
 import io.reactivex.schedulers.TestScheduler
 import org.mockito.Mockito
 
-class TestPresenterConfiguration private constructor(userSettings: UserSettings, restApi: RestApi, crashReporter: CrashReporter)
-    : PresenterConfiguration(TestScheduler(), TestScheduler(), userSettings, restApi, crashReporter) {
+class TestViewModelConfiguration private constructor(userSettings: UserSettings, restApi: RestApi, timeProvider: TimeProvider, crashReporter: CrashReporter)
```

Comment:
Probably out-of-scope for this PR but I don't really know that there's much value to keeping a separate `TestViewModelConfiguration` class. I think it'd be cleaner to just have a top-level `createTestConfiguration()` function that creates a `ViewModelConfiguration` instance with test dependencies.

Reply:
the main advantage of this class is that it auto casts the ioScheduler and uiScheduler into TestScheduler